### DISCUSSION TITLE
Welcome to reacher Discussions!

### DISCUSSION THREAD

**Discussion by strikewalker** [strikewalker/reacher#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by climardo** [comment]
Cool project. Now that Jack blew this up, I'm betting you'll get a bunch of feature requests and questions. I'll check the code out later when I'm on my computer; thanks for making this open source!
